client/coral-embed-stream/src/tabs/stream/components/EditableCommentContent.js

```diff
@@ -72,8 +74,14 @@ export class EditableCommentContent extends React.Component {
     }
   }
 
-  handleBodyChange = body => {
-    this.setState({ body });
+  handleBodyChange = (body, ...data) => {
```

Comment:
Hmm weird, an array of arguments is allowed but then we just take the first one. How about just `handleBodyChange = (body, data)`?